rules/cre/annotation_mosaic.smk

```diff
@@ -0,0 +1,66 @@
+rule vep_mosaic:
+    input:
+        "filtered/{family}-freebayes_mosaic.uniq.normalized.decomposed.vcf.gz"
+    output:
+        temp("annotated/mosaic/vep/{family}_mosaic.coding.vep.vcf")
+    log:
+        "logs/vep/{family}_mosaic.vep.coding.log"
+    threads: 10
+    resources:
+        mem_mb = 30000
+    params:
+        dir=config["annotation"]["vep"]["dir"],
+        dir_cache=config["annotation"]["vep"]["dir_cache"],
+        ref=config["ref"]["genome"],
+    wrapper:
+        get_wrapper_path("vep")
+
+rule vcfanno_mosaic:
+    input:
+        "annotated/mosaic/vep/{family}_mosaic.coding.vep.vcf"
+    output:
+        "annotated/mosaic/vcfanno/{family}_mosaic.coding.vep.vcfanno.vcf"
+    log:
+        "logs/vcfanno/{family}_mosaic.vcfanno.coding.log"
+    threads: 10
+    resources:
+        mem_mb = 20000
+    params:
+        lua_script = config["annotation"]["cre.vcfanno"]["lua_script"],
+       	conf = config["annotation"]["cre.vcfanno"]["conf"],
+        base_path = config["annotation"]["cre.vcfanno"]["base_path"],
+    wrapper:
+        get_wrapper_path("vcfanno")
+
+rule fix_dp_mosaic:
+    input:
+        "annotated/mosaic/vcfanno/{family}_mosaic.coding.vep.vcfanno.vcf"
+    output:
+        "annotated/mosaic/vcfanno/{family}_mosaic.coding.vep.vcfanno.wDP.vcf"
+    threads: 1
+    resources:
+        mem_mb = 20000
+    shell:
+        '''
+        export _JAVA_OPTIONS=\"-Xmx2048m\"
+        rtg vcfsubset -i {input} -o annotated/mosaic/vcfanno/noDP.vcf.gz --remove-info DP
```

Comment:
Can you please make a conda environment for this rule that includes rtg and bcftools? Calling rtg like this means that rtg must be in the users path i.e. installed separately. This makes it difficult when implementing the pipeline elsewhere e.g. the CHEO-RI space, because there are undeclared dependencies.